doc/dnf5_plugins/repoclosure.8.rst

```diff
@@ -22,14 +22,14 @@
  Repoclosure Command
 ####################
 
-Display a list of unresolved dependencies for repositories.
-
 Synopsis
 ========
 
 ``dnf5 repoclosure [options] [<pkg-spec>...]``
 
 
+Description
+===========
 
 The ``repoclosure`` command in DNF5 allows you to analyze package metadata from multiple repositories. It checks all dependencies of the packages and provides a list of packages that have unresolved dependencies.
```

Comment:
What about
"The ``repoclosure`` command allows you to analyze package metadata from multiple repositories. It checks all dependencies of the packages and provides a list of packages that have unresolved dependencies."

Reply:
Changed.